x-pack/filebeat/module/checkpoint/firewall/config/firewall.yml

```diff
@@ -3,6 +3,22 @@
 type: syslog
 protocol.udp:
   host: "{{.syslog_host}}:{{.syslog_port}}"
+  {{ if ne .pipeline "" }}
```

Comment:
What is `pipeline` used for?

Reply:
No, but it still is nice to be able to configure it.